bluemira/equilibria/coils/_grouping.py

```diff
@@ -998,3 +996,18 @@ def _sum(
         ind = self._control_ind if control else slice(None)
 
         return np.sum(output[..., ind], axis=-1) if sum_coils else output[..., ind]
+
+    @classmethod
+    def from_group_vecs(cls, eqdsk: EQDSKInterface):
+        """Same as its parent class,
+        initialize an instance of CoilSet from group vectors,
+        but it also auto-populate the control attribute.
+
+        This is done so that the user doesn't have to
+        manually set passive coils as non-controlled.
+        """
+        self = super().from_group_vecs(eqdsk)
```

Comment:
Can you double check for me that the type of self here is `Coilset`?  Possibly add a test to that effect

Reply:
Change applied, test written.